### PR TITLE
chore: enroll worldchain, xlayer, cheesechain, zircuit validators

### PR DESCRIPTION
**Pull request by paulbalaji**
- chore: enroll worldchain, xlayer, cheesechain, zircuit validators
- fix: update cosmos zod schema
- drive-by: update core checker to check if the ism in registry matches the expected config

### CODE REVIEW

**Review by tkporter — APPROVED** [review]

File: typescript/sdk/src/utils/cosmos.ts [hunk 1]
```diff
@@ -431,18 +741,22 @@ export const CosmosChainSchema = z
...
+            block_page: z.string().min(1).optional(),
           })
           .strict(),
       )
       .optional(),
-    keywords: z.array(z.string()).optional(),
+    keywords: z.array(z.string().min(1)).optional(),
     extra_codecs: z.array(z.enum(['ethermint', 'injective'])).optional(),
   })
-  .passthrough()
```
Comment (tkporter) [inline typescript/sdk/src/utils/cosmos.ts:445]: can you keep `passthrough()`, we intentionally added this to be robust to schema additions on their side

**Review by tkporter — APPROVED** [review]

File: typescript/sdk/src/utils/cosmos.ts (hunk 1 above)
Comment (tkporter) [reply typescript/sdk/src/utils/cosmos.ts:445]: a comment about this would help too

Comment (paulbalaji) [reply typescript/sdk/src/utils/cosmos.ts:445]: spoke offline but 

passthrough vs strict.and(intersection(any,any)) should be the same thing, will leave a comment

File: typescript/sdk/src/consts/multisigIsm.ts [hunk 2]
```diff
@@ -518,8 +518,13 @@ export const defaultMultisigConfigs: ChainMap<MultisigConfig> = {
   },
 
   zircuit: {
-    threshold: 1,
-    validators: ['0x169ec400cc758fef3df6a0d6c51fbc6cdd1015bb'],
+    threshold: 2,
```
Comment (paulbalaji) [inline typescript/sdk/src/consts/multisigIsm.ts:521]:
```suggestion
    threshold: 3,
```

**Pull request merged**